bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/editor/SummernoteEditorCssReference.java

```diff
@@ -0,0 +1,27 @@
+package de.agilecoders.wicket.extensions.markup.html.bootstrap.editor;
+
+import org.apache.wicket.request.resource.CssResourceReference;
+
+public class SummernoteEditorCssReference extends CssResourceReference {
```

Comment:
The idea of Webjars is that the application can provide custom version of the JS widget (newer or older one) by adding Maven dependency to its pom.xml (same for Gradle, SBT, ...).
By putting the resource in the classpath we should either add custom code to be able to set custom ResRef or one has to deal with ClassLoader ordering to provide custom version of the .js/.css resource.